src/lib.rs

```diff
@@ -716,6 +758,35 @@ impl<T> Slab<T> {
         }
     }
 
+    /// Return two mutable references to the values associated with the two
+    /// given keys simultaneously without performing bounds checking.
+    ///
+    /// This function should be used with care.
```

Comment:
A `# Safety` section documenting the invariants that the caller must uphold (such as key1 != key2) would be nice.